lib/internal/process/stdio.js

```diff
@@ -131,7 +131,7 @@ function createWritableStdioStream(fd) {
       break;
 
     case 'FILE':
-      var fs = require('fs');
+      var fs = require('internal/fs');
```

Comment:
I'd rather not in this PR. There are several similar updates that can be made elsewhere in this file that should be done in a separate PR